sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosStoredProcedureResponse.java

```diff
@@ -90,13 +94,56 @@ public double getRequestCharge() {
         return super.getRequestCharge();
     }
 
+    @Override
+    public String getMaxResourceQuota() {
```

Comment:
with this change essentially similar to `CosmosItemResponse`  we are implementing all these APIs here instead of relying on the parent class.

So I would like to suggest that we drop the inheritance on the `CosmosResponse` for this class.
I know that theoretically speaking it can be considered a breaking change but in practice it is not breaking change as you are implementing all the required methods.